types/react-router-bootstrap/lib/LinkContainer.d.ts

```diff
@@ -1,17 +1,17 @@
-import { ComponentClass, ReactNode } from 'react';
+import type { ComponentClass, CSSProperties, MouseEventHandler, ReactNode } from 'react';
 
 // See https://github.com/react-bootstrap/react-router-bootstrap/blob/master/src/LinkContainer.js
 interface LinkContainerProps {
     children: ReactNode;
-    onClick?: () => void;
+    onClick?: MouseEventHandler<HTMLElement>;
     replace?: boolean;
-    to: string | object;
+    to: string | { pathname: string };
     state?: object;
     className?: string;
     activeClassName?: string;
-    style?: string | number;
-    activeStyle?: string | number;
-    isActive?: () => void | boolean;
+    style?: CSSProperties;
+    activeStyle?: CSSProperties;
+    isActive?: (match: any, location: any) => boolean | boolean;
```

Comment:
I'm not sure the `boolean | boolean` makes much sense?

Reply:
Whoops! You're right, I need an extra pair of parentheses here.